src/back/msl.rs

```diff
@@ -74,12 +70,17 @@ enum LocationMode {
     Uniform,
 }
 
+#[derive(Debug, Clone, Copy)]
```

Comment:
Not entirely sure about this, it also causes clippy to get angry with `resolve_binding` as it's cheaper to take `self` rather than `&self` there now that it is copy.